pkgs/games/vvvvvv/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, fetchurl, fetchFromGitHub, requireFile, makeWrapper
+, SDL2, SDL2_mixer, cmake, ninja
+, Foundation
+, fullGame ? false }:
+
+let
+  dataZip = if fullGame then requireFile {
+    # the data file for the full game
+    name = "data.zip";
+    sha256 = "1q2pzscrglmwfgdl8yj300wymwskh51iq66l4xcd0qk0q3g3rbkg";
+    message = ''
+      In order to install VVVVVV, you must first download the game's
+      data file (data.zip) as it is not released freely.
+      Once you have downloaded the file, place it in your current
+      directory, use the following command and re-run the installation:
+      nix-prefetch-url file://\$PWD/data.zip
+    '';
+  } else fetchurl {
+    # the data file for the free Make and Play edition
+    url = https://thelettervsixtim.es/makeandplay/data.zip;
```

Comment:
Is there a versioned link?

Reply:
No.

Reply:
`data.zip` has never changed since it was first introduced in 2014. Due to compatibility and the fact that this is an 11-year-old game, it probably won't ever be changed either. (But new zips will probably be introduced.)